File: 16000-16999/16785_Coin/coin.cpp
#include    <iostream>

using namespace std;

#define DAYS_PER_WEEK   7

int main(void)
{
    cin.tie(NULL);
    cin.sync_with_stdio(false);

    int A,B,C,d,coin;

    cin>>A>>B>>C;

    for(d=1,coin=0;;d++)
    {
        coin += A+(!(d%DAYS_PER_WEEK)?B:0);
        if(coin>=C) break;
    }

    cout<<d<<'\n';
    return  0;
}

File: 16000-16999/16785_Coin/coin.py
A,B,C = input().split()

A = int(A)
B = int(B)
C = int(C)

d = 1
coin = 0

while(True) :
    coin += A
    if d%7 == 0 :
        coin += B
    if coin >= C :
        break
    d += 1

print(d)